js/main.js

```diff
@@ -53,6 +53,7 @@
         pulsar.tableDetail.init();
         pulsar.dropZoneComponent = pulsar.DropZoneComponentFactory.create($('body')[0], '.dropzone');
         pulsar.repeaterManager.init();
+        pulsar.tooltipFactory($html);
```

Comment:
so here we just need to call our factory method to get our instance and then call init on the instance

Reply:
so this might look something like...

```
// We'll put the tooltip listener on pulsar's global object here...
pulsar.tooltipListener = pulsar.tooltipFactory($html);

// Then invoke it's init method
pulsar.tooltipListener.init();
```
```